aws/ec2/resources.py

```diff
@@ -135,3 +137,14 @@ def ec2_security_groups_with_in_use_flag():
             sec_group["InUse"] = False
 
     return sec_groups
+
+
+def ec2_images_owned_by():
+    "Returns a list of EC2 images owned by a list of provided accounts"
+    return (
+        botocore_client.get("ec2", "describe_images", [],
+                            {"Filters": [{"Name": "owner-id", "Values": ["361527076523", "351644144250", "178550527431", "342354050843", "927034868273"]}]})
```

Comment:
we need to get that list from a config file somehow. @ajvb is that something you could quickly add to this patch?

Also, I tried adding a long list of account IDs here, and it seems to make pytest fail to create cache files because it tried to put all the IDs in the filename.

Reply:
Yeah absolutely. Do we actually want more account IDs than the one we are scanning against?